recipes-devtools/firmware-package-tool/firmware-package-tool.bb

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) Siemens AG, 2023
+#
+# Authors:
+#  Li Hua Qian <huaqian.li@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+DESCRIPTION = "Generate The Firmware Update Package"
+MAINTAINER = "huaqian.li@siemens.com"
+
+SRC_URI = "file://update.conf.json \
+           file://iot2050-fwu-update-json.sh"
+
+inherit dpkg-raw
+
+do_deploy_deb[nostamp] = "1"
```

Comment:
Why "nostamp"?

Reply:
I was thinking that it is good to generate this tarball each building.

Reply:
...which is a bug.